src/worker-type.js

```diff
@@ -779,6 +780,18 @@ WorkerType.prototype.determineCapacityChange = function(runningCapacity, pending
   return capacityChange;
 };
 
+WorkerType.prototype.declareWorkerType = function({queue}) {
+  // keep the worker around well over 24 hours, as we will likely re-declare
+  // at least that often
+  const expires = taskcluster.fromNow('36 hours');
```

Comment:
Why is there an expires field?  We derive that information from the worker type being seen or not in the queue.

Reply:
Because the workerType might not be seen if no workers are provisioned.